src/components/SimpleTable.tsx

```diff
@@ -125,49 +28,93 @@ export interface IProps {
-        </Cell>
-      </Row>
-    )}
-    {!hasError && content.length === 0 && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-loading" />
-        </Cell>
-      </Row>
-    )}
-    {content.map((row, idx) => {
-      const { name, marketCap, image, type, url } = row
-      const rowContent = (
-        <>
-          <Cell>
-            {image && <StyledImage src={image} alt="" />}
-            {name}
-          </Cell>
-          <Cell>{marketCap}</Cell>
-          <Cell>{type}</Cell>
-        </>
-      )
+const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
+  const [textColor] = useToken("colors", ["text"])
 
-      return url ? (
-        <LinkRow key={idx} to={url}>
-          {rowContent}
-        </LinkRow>
-      ) : (
-        <Row key={idx}>{rowContent}</Row>
-      )
-    })}
-  </Table>
-)
+  return (
+    <Table
+      variant="unstyled"
+      my={8}
+      borderRadius="2px"
+      border={`1px solid ${textColor}`}
+      bg="background"
+      mb={8}
+      minW="720px"
+    >
+      <Thead bg="ednBackground" color="text200">
+        <Tr boxShadow={`0 1px 1px ${textColor}`} mb="1px">
+          {columns.map((column, idx) => (
+            <Th
+              key={idx}
+              py={5}
+              fontSize="md"
+              fontWeight="normal"
+              letterSpacing="normal"
+            >
+              {column}
+            </Th>
+          ))}
 
+          {content && content[0]?.url && (
+            <Th p={5} fontSize="md" fontWeight="normal" textAlign="end">
+              ↗
+            </Th>
+          )}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {hasError && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-error" />
+            </Td>
+          </Tr>
+        )}
+        {!hasError && content.length === 0 && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-loading" />
+            </Td>
+          </Tr>
+        )}
+
+        {content.map(({ name, marketCap, image, type, url }, idx) => (
+          <Tr
+            as={url ? Link : Tr}
+            display="table-row"
+            href={url}
+            key={idx}
+            color="white"
+            textDecoration="none"
+            boxShadow={`0 1px 1px ${textColor}`}
+            _hover={{
+              textDecoration: "none",
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+              cursor: url ? "pointer" : "default",
+            }}
+            _focus={{
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+            }}
+          >
+            <Td display="flex" alignItems="center">
+              {image && <Image src={image} alt="" mr={4} boxSize={6} />}
+              {name}
```

Comment:
Provide `LinkOverlay` here and using the custom `Link` component instead of the Chakra version, with necessary styling overrides. (I like `inherit` here since most styling happens in the table components)

Also ensure that the overlay does not exist if `url` is undefined, else styling bugs can occur, or the anchor tag with no href will still exist (which is not accessible).
```suggestion
              { url ? (
                <LinkOverlay
                  as={Link}
                  hideArrow
                  to={url}
                  color="inherit"
                  textDecoration="inherit"
                  _hover={{
                    textDecoration: "inherit",
                  }}
                >
                  {name}
                </LinkOverlay>
              ) : (
                <>{name}</>
              )}
```

Reply:
Yeah, I also thought of using the `LinkOverlay` component, but can't figure out how to use it in this scenario. Thanks for the help, I made all the changes as you said.

Reply:
Thats a great idea @TylerAPfledderer thanks for mentioning that!